Dockerfile

```diff
@@ -170,6 +171,15 @@ RUN wget ${WGET_ARGS} -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key
 	apt-get update && \
 	apt-get install -y clang-$LLVM_VERSION lldb-$LLVM_VERSION lld-$LLVM_VERSION clangd-$LLVM_VERSION llvm-$LLVM_VERSION-dev
 
+# Install sparse package for static analysis
+RUN mkdir -p /opt/sparse && \
+	cd /opt/sparse && \
+	git clone https://git.kernel.org/pub/scm/devel/sparse/sparse.git && \
+	cd sparse && git checkout ${SPARSE_COMMIT} && \
```

Comment:
```suggestion
	cd sparse && git checkout ${SPARSE_VERSION} && \
```

Reply:
@stephanosio Yes, we need the feature from the development branch. I understand that there are sparse tar.gz releases, but the latest sparse  0.6.4 doesn't contain the feature we need. that's why I have to use the development branch and specify a commit ID.
But I can promise you that once there is a new release or a new tag that covers the feature we need, I will submit a new PR.

Reply:
updated, thank you